react-calculator/src/components/Calculator.js

```diff
@@ -6,15 +6,19 @@ const Calculator = () => {
   const [operation, setOperation] = useState('+')
   const [operand2, setOperand2] = useState('')
   const [answer, setAnswer] = useState('')
+  const [error, setError] = useState('')
     
 
   const upDateNumber = (event) => {
       setOperand1(event.target.value)
   }
 
   const calculate = () =>{
-
+      //if (!calculate()isValid){setAnswer('')
```

Comment:
const calculateAnswer = () => {
  if(isNaN(operand1) || isNaN(operand2)) {
    return setError("Please provide a valid number for both operands")
  } else {
     setAnswer('')
     setError('')